src/Eloquent/Model.php

```diff
@@ -155,7 +155,7 @@ public function getAttribute($key)
         }
 
         // Dot notation support.
-        if (Str::contains($key, '.') && Arr::has($this->attributes, $key)) {
+        if (str_contains($key, '.') && Arr::has($this->attributes, $key)) {
```

Comment:
`$key` is a string.

Reply:
Based on https://laravel.com/docs/10.x/helpers#method-str-contains, am I correct to assume that `Str::contains()` only differs in its handling of array parameters?